src/Main.purs

```diff
@@ -23,15 +25,20 @@ import Effect.Exception (catchException, throw, throwException)
 import Effect.Now (now)
 import Effect.Ref as Ref
 import Foreign.Object as FO
-import Node.ChildProcess as Child
+import Node.Buffer as Buffer
+import Node.Encoding (Encoding(..))
 import Node.Encoding as Encoding
+import Node.Errors.SystemError (code, toError)
+import Node.EventEmitter as H
 import Node.FS.Stats as Stats
 import Node.FS.Sync as File
 import Node.Path as Path
 import Node.Platform (Platform(Win32))
+import Node.Process (stderr, stdout)
 import Node.Process as Process
-import Node.Stream as Stream
-import Partial.Unsafe (unsafePartial)
+import Node.Stream (dataH)
+import Node.UnsafeChildProcess.Unsafe (spawn', unsafeStderr, unsafeStdout) as UnsafeChild
+import Node.UnsafeChildProcess.Safe (errorH, exitH) as UnsafeChild
```

Comment:
Can you clarify what UnsafeChildProcess is about and why it's needed? I'm not up to date on everything that happened with childprocess.

Reply:
In the latest Node.ChildProcess, appendStdio is there in spawn', which appends stdio to [pipe, pipe, pipe], causing issues with seeing the "Compiling Module" message.

So, I'm using Node.UnsafeChildProcess.Unsafe.spawn' instead:

```
let stdio =
      case outputStream of
        Stdout -> [pipe, pipe, shareStream stderr]
        Stderr -> [pipe, shareStream stdout, pipe]
```
This way, the streams are configured correctly. Here's the [PursuitLink](https://pursuit.purescript.org/packages/purescript-node-child-process/11.1.0/docs/Node.ChildProcess).